.travis.yml

```diff
@@ -4,6 +4,7 @@ rvm:
   - 2.1.5
   - 2.2.0
   - 2.3.0
+before_install: gem install bundler -v 1.14.1
```

Comment:
minor: let's put this line between line number 15 and 16 to have one section of `before_install`